bin/MadGraph5_aMCatNLO/cards/production/8TeV/WJToLNu_NLO/wellnu2j_5f_NLO_FXFX_proc_card.dat

```diff
@@ -0,0 +1,15 @@
+import model loop_sm-ckm_no_b_mass
+#switch to diagonal ckm matrix if needed for speed
+#import model loop_sm-no_b_mass
+
+define p = p b b~
+define j = j b b~
+
+define ell+ = e+ mu+ ta+
+define ell- = e- mu- ta-
+
+generate p p > ell+ vl jj $$ t t~ h [QCD] @0
+add process p p > ell- vl~ jj $$ t t~ h [QCD] @1
+
+output wellnu012j_5f_NLO_FXFX -nojpeg
```

Comment:
in general the name of the output needs to match the name of the cards.  eg this should be
"output wellnu2j_5f_NLO_FXFX -nojpeg"